addons/resource/migrations/12.0.1.1/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,14 @@
+---Fields in module 'resource'---
+resource     / resource.calendar        / hours_per_day (float)         : NEW 
+resource     / resource.calendar.attendance / day_period (selection)        : NEW required: required, selection_keys: ['afternoon', 'morning'], req_default: function
+resource     / resource.calendar.leaves / time_type (selection)         : NEW selection_keys: ['leave', 'other']
+# NOTHING TO DO
+
+resource     / resource.calendar.leaves / tz (selection)                : DEL selection_keys: function
+resource     / resource.calendar        / tz (selection)                : NEW required: required, selection_keys: function, req_default: function
+# Done: post-migration: filled tz from leave_ids.tz
+
+resource     / resource.resource        / tz (selection)                : NEW required: required, selection_keys: function, req_default: function
+# Done: post-migration: filled tz from user_id.tz
```

Comment:
There are other possible resources to take into account when doing the migration scripts. For example, workcenters for `mrp` module. But this is to be done in that module :wink:

Reply:
And thus I should do something?